apps/mymedicare_cb/models.py

```diff
@@ -108,7 +174,7 @@ def create_beneficiary_record(username=None,
                                  user_id_type=user_id_type)
 
         # Extra user information
-        # TODO: remvoe the idea of UserProfile
+        # TODO: remove the idea of UserProfile
```

Comment:
😆

Reply:
@karlmdavis For your question about the need for a transaction, this is OK. The `crosswalk.save()` method call will update an existing Crosswalk entry. Related to lines 62-64 and 69-70 those will update the fields for an existing Crosswalk record in the DB.   

In the other case after the `except User.DoesNotExist`, where the User/Crosswalk don't exist yet, the creation of the new model records are grouped in to a transaction using `with transaction.atomic()` in the `create_beneficiary_record()` function.

Reply:
That makes sense to me. Apologies for not grokking that, myself.